shaders/glow.shader

```diff
@@ -13,6 +14,10 @@ void vertex()
 	// increase the size by the the width, this extra space is needed for the outline
 	extra_movement = TEXTURE_PIXEL_SIZE * width * 2.;
 	VERTEX *= 1. + extra_movement;
+	if (recenter)
+	{
+		VERTEX -= vec2(1, 1) * width;
```

Comment:
I'm pretty sure the syntax `vec2(width)` works in glsl

Reply:
True, I'll fix that